src/server/saved_objects/client/lib/create_find_query.js

```diff
@@ -35,5 +40,20 @@ export function createFindQuery(options = {}) {
     });
   }
 
-  return { version: true, query: { bool } };
+  const query = { version: true, query: { bool } };
+
+  if (sortField) {
+    const value = {
+      order: sortOrder,
+      unmapped_type: get(mappings, `${type}.properties.${sortField}.type`)
```

Comment:
Are you sure these mapping paths will change for v6? As far as I can tell, these mappings are being specified by https://github.com/elastic/kibana/blob/master/src/core_plugins/kibana/mappings.json and I didn't see any changes to this for v6.